readme.md

```diff
@@ -9,11 +9,13 @@ For current progress on Dojo 2, contribution guidelines and other information, p
 There are currently three examples:
 
 - [todo-mvc](./todo-mvc) - A reference implementation of [Todo MVC](http://todomvc.com/) built in current pre-release versions of Dojo 2 packages.
+- [todo-mvc-kitchensink](./todo-mvc-kitchensink) - A feature-rich version of todo-mvc, built in current pre-release versions of Dojo 2 packages.
```

Comment:
let's be consistent throughout and call it TodoMVC, since that's what it is called at todomvc.com